internal/pkg/mapper/defaultbucket/defaultbucket.go

```diff
@@ -0,0 +1,50 @@
+package defaultbucket
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+type defaultBucketMapper struct {
+	model.MapperContext
+}
+
+func NewMapper(context model.MapperContext) *defaultBucketMapper {
+	return &defaultBucketMapper{MapperContext: context}
+}
+
+func (m *defaultBucketMapper) visitStorageInputTables(config *model.Config, callback func(config *model.Config, inputTableSource string, inputTable *orderedmap.OrderedMap) error) error {
+	inputTablesRaw, found, err := config.Content.GetNested("storage.input.tables")
+	if !found {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
+	inputTables, ok := inputTablesRaw.([]interface{})
+	if !ok {
+		return nil
```

Comment:
Toto je nekonzistentne, ... ak bude mat napr. `storage` nespravny typ, tak vyssie v kode vratis `return err`, ... ale ak bude mat table nespravny typ, tak to ignorujes.

Ak nas zaujima iba `[]interface{}` a ak kluc neexistuje, alebo ma iny typ -> tak ho preskocime.
Tak to staci zapisat takto:

```go
inputTablesRaw, _, _ := config.Content.GetNested("storage.input.tables")
inputTables, ok := inputTablesRaw.([]interface{})
if !ok {
	return nil
}
```

Reply:
Sorry ak som ta pomylil ja, ... riesil som iba tie padajuce testy, ... nepozeral som, co dalej v kode nasleduje.

Reply:
V pripade chyby bude totiz `inputTablesRaw`  -> `nil`, takze `err` sa uz nemusi kontrolovat.